gen/dcos-config.yaml

```diff
@@ -629,6 +629,9 @@ package:
       {
         "role": "agent_public"
       }
+  - path: /etc/dcos-ui-service.env
+    content: |
+      DCOS_UI_SERVICE_CONFIG_PATH=/opt/mesosphere/etc/dcos-ui-service-config.json
```

Comment:
Is the assumption that the UI-service reads from a config file? AFAIK this is not implemented, but we rather get our information from command line flags.

Reply:
I don't think we plan on implementing it

Reply:
Okay, then I will consider how we can read from command line flags here as well.